frontend/src/metabase/app.js

```diff
@@ -135,6 +143,29 @@ angular.module('metabase', [
     $routeProvider.when('/admin/people/', route);
     $routeProvider.when('/admin/settings/', { ...route, template: '<div class="full-height" mb-redux-component />' });
 
+    $routeProvider.when('/reference', route);
```

Comment:
I'm pretty sure we can leave out any routes that use the default `route` object since it would be handled by `$routeProvider.otherwise(route);`. It's only when we need to provide a custom `redirectTo` or `template` that we need to specify this. Eventually they'll all go away.

If you verify everything works as expected with them removed go ahead and remove these (and add a comment to this effect at the top, if you don't mind)